internal/engine/ingester/diff/diff.go

```diff
@@ -216,7 +216,7 @@ func ingestFileForFullDiff(filename, patch, patchUrl string) (*pb.PrContents_Fil
 		} else if strings.HasPrefix(line, "+") {
 			result = append(result, &pb.PrContents_File_Line{
 				Content:    line[1:],
-				LineNumber: int32(currentLineNumber),
+				LineNumber: currentLineNumber,
```

Comment:
We could just declare `currentLineNumber` as int32 here on line 206, particularly since we parse it as an int32 on line 212.

Reply:
Tried this, but since `strconv.ParseInt` returns `(int64, error)` we can't just wrap the return value in `int32()`.